integration/helpers/vite-template/.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 /.cache
 /build
 .env
+.react-router
```

Comment:
This isn't strictly needed, but it's here for completeness since it's in all other test templates and it makes the test template more aligned with our official templates.